typescript/cosmos-sdk/scripts/run-e2e-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+function cleanup() {
+  docker compose down
+}
```

Comment:
This is not something that you need to do in this PR, but it might be worth exploring later. This allows to control docker containers from nodejs https://node.testcontainers.org/

Reply:
nice, will try implement it